code/modules/projectiles/projectile/special.dm

```diff
@@ -55,8 +55,7 @@
 
 /obj/item/projectile/temp/New(loc, shot_temp)
 	..(loc)
-	if(shot_temp)
-		temperature = shot_temp
+	temperature = shot_temp

```

Comment:
Should probably be:
```
if(!isnull(shot_temp))
	temperature = shot_temp
```
Otherwise it will runtime if New() is ever called without a specific temperature, which is something that from my testing *did* happen, although I can't remember specifically when/why.